src/argilla/server/contexts/datasets.py

```diff
@@ -264,6 +265,19 @@ async def create_metadata_property(
     return metadata_property
 
 
+async def update_metadata_property(
+    db: "AsyncSession",
+    metadata_property: MetadataProperty,
+    metadata_property_update: MetadataPropertyUpdate,
+):
+    # TODO: Do we need some additional step to reindex on the search engine?
```

Comment:
@frascuchon the question in the TODO is for you. 🥲

Reply:
No search engine reindex required since the "updatable" attributes does not affect to the search index